server/botserver.py

```diff
@@ -101,7 +102,8 @@ def run(self):
         while self.running:
             try:
                 self.load_config()
-                self.slack_wrapper = SlackWrapper(self.get_config_option('api_key'))
+                self.slack_wrapper = SlackWrapper(
+                    self.get_config_option('api_key'))
```

Comment:
Use `"` like everywhere else.

Reply:
This is just a reformat?

Reply:
Yes, but might as well correct if you're editing the line.